src/go/rpk/pkg/cli/redpanda/mode.go

```diff
@@ -50,7 +50,7 @@ func NewModeCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 		Run: func(_ *cobra.Command, args []string) {
 			err := executeMode(fs, p, args[0])
 			out.MaybeDieErr(err)
-			fmt.Printf("Successfully set mode to %q.", args[0])
+			fmt.Printf("Successfully set mode to %q.\n", args[0])
```

Comment:
[question] Should we check if Redpanda is running and, if necessary, warn the user that a restart is needed for any changes in the redpanda.yaml to take effect?

Reply:
Might make more sense to have a --restart flag + a comment that config changes won't apply until after a restart in the docs?